proto/controls/common/v1/device.proto

```diff
@@ -86,5 +122,7 @@ message Value {
         AnalogAlarm anaAlarm = 6;
         DigitalAlarm digAlarm = 7;
         BasicStatus basicStatus = 8;
+        Alarm alarm = 9;
+        AlarmArray alarmArr = 10;
```

Comment:
I don't know what the use case for multiple alarms is. Is this is related to the FSM discussion?
Honestly, I like this idea, but I wonder if this idea is duplicative of the hi, hihi situation. A problem for future us.

Reply:
I was also wondering if it should just be an array of alarms and many times the array has one element (i.e. drop line 125.)